src/components/api-request.js

```diff
@@ -500,7 +500,9 @@ export default class ApiRequest extends LitElement {
             <div class="param-name ${fieldSchema.deprecated ? 'deprecated' : ''}">
               ${fieldName}${!fieldSchema.deprecated && (schema.required && schema.required.includes(fieldName) || fieldSchema.required) ? html`<span style='color:var(--red);'>*</span>` : ''}
             </div>
-            <div class="param-type">${paramSchema.type}</div>
+            <div class="param-type">
+            ${paramSchema.type === 'array' ? html`[<span class="formnum">${paramSchema.format}</span>]` : `${paramSchema.type}`}
```

Comment:
If it is an array of booleans, I don't see this working correctly, since you are making a similar change in the schema-tree, I would normally expect a similar solution here.